api/src/main/java/org/openmrs/api/db/ConceptDAO.java

```diff
@@ -326,55 +366,139 @@ public Integer getCountOfConcepts(String phrase, List<Locale> locales, boolean i
 	public List<Concept> getConceptsByMapping(String code, String sourceName, boolean includeRetired);
 	
 	/**
+	 * Get Concept by its UUID
+	 * 
 	 * @param uuid
 	 * @return concept or null
+	 * @should find object given valid uuid
+	 * @should return null if no object found with given uuid
 	 */
 	public Concept getConceptByUuid(String uuid);
 	
 	/**
+	 * Get ConceptClass by its UUID
+	 * 
 	 * @param uuid
 	 * @return concept class or null
+	 * @should find object given valid uuid
+	 * @should return null if no object found with given uuid
 	 */
 	public ConceptClass getConceptClassByUuid(String uuid);
 	
+	/**
+	 * Get ConceptAnswer by its UUID
+	 * 
+	 * @param uuid
+	 * @return concept answer or null
+	 * @should find object given valid uuid
+	 * @should return null if no object found with given uuid
+	 */
```

Comment:
Since this method is well documented in the `ConceptService` interface, don't you wanna reference to that documentation with the help of `@see` javadoc attribute just like its done in other places in this class?